README.md

```diff
@@ -60,6 +60,13 @@ Cache directroy (only effective if `cache` is `true`)
 
 Transform function. See [src/babel](./src/babel.ts) for more details
 
+### `transformPreset`
+
+- Type: String
```

Comment:
If adding a new option, what do you think of supporting `['esbuild', { async: false }]`?